package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "pretest": "cross-env NODE_ENV=test npm run migrate:undo && cross-env NODE_ENV=test npm run migrate && cross-env NODE_ENV=test npm run seed",
```

Comment:
This change is not supposed to be on this route

Reply:
I thought there would be a need to implement it because of my tests. I needed to undo and re-run migrations and then seed the db so that I can check if a user exists.